saleor/graphql/account/tests/test_account_handle_invalid_fields.py

```diff
@@ -0,0 +1,55 @@
+import pytest
```

Comment:
Do we need separate file for those tests? Shouldn't this be part of `test_account_address_create`?

Reply:
@kadewu I created a separate test file for easy review. However, I'm open to adding the tests to the main test file if you prefer.

Reply:
There is no reason to create separate file for just review purpose, as I believe there is no difference anyway. If that was the only reason, I would suggest to move the tests to the proper test file. It may be also a good idea then to follow common pattern for testing mutations.

Reply:
Thanks for the feedback. Working on this.